src/components/material/material-buttons/physical/MaterialButtonsPhysical.tsx

```diff
@@ -41,7 +41,6 @@ const MaterialButtonsPhysical: React.FC<MaterialButtonsPhysicalProps> = ({
     return <MaterialButtonUserBlocked size={size} />;
   }
 
-  // TODO: Investigate if we could use UseReservableManifestations() instead.
   if (!areAnyReservable(data)) {
```

Comment:
Yes, I have investigated. I think something is wrong with that hook, because it does exactly the opposite of what one would expect - showing that there are reservable manifestations when the libraries don't have any (find on shelf) AND the availability labels show that there aren't any available.